libcob/call.c

```diff
@@ -83,11 +83,12 @@ lt_dlsym (HMODULE hmod, const char *p)
 	return modun.voidptr;
 }
 
-#define lt_dlopenlcl(x)	lt_dlopen(x)
-#define lt_dlclose(x)	FreeLibrary(x)
+#define lt_dlopenlcl(x)		lt_dlopen(x)
+#define lt_dlclose(x)		FreeLibrary(x)
 #define	lt_dlinit()
 #define	lt_dlexit()
-#define lt_dlhandle	HMODULE
+#define lt_dlhandle		HMODULE
+#define lt_dladdsearchdir(x)	AddDllDirectory(x)
```

Comment:
that's a WIN32 definition and we don't use that for this case - you can therefore revert those lines